homeassistant/components/invoxia/device_tracker.py

```diff
@@ -0,0 +1,138 @@
+        GpsTrackerEntity(coordinator, client, tracker)
+        for tracker, coordinator in zip(trackers, coordinators)
+    ]
+
+    hass.data[DOMAIN][entry.entry_id][CONF_ENTITIES].extend(entities)
+    async_add_entities(entities, update_before_add=True)
+
+
+@dataclass(frozen=True)
+class GpsTrackerEntityDescription(EntityDescription):
+    """Describes a Invoxia GPS tracker entity."""
+
+
+class GpsTrackerEntity(
+    CoordinatorEntity[GpsTrackerCoordinator], device_tracker.TrackerEntity
+):
+    """Class for Invoxia™ GPS tracker devices."""
+
+    entity_description: GpsTrackerEntityDescription
+
+    _attr_attribution = ATTRIBUTION
+    _attr_should_poll = False
+
+    def __init__(
+        self, coordinator: GpsTrackerCoordinator, client: AsyncClient, tracker: Tracker
+    ) -> None:
+        """Store tracker main properties."""
+        super().__init__(coordinator)
+
+        # Attributes for update logic
+        self._client: AsyncClient = client
+        self._tracker: Tracker = tracker
+
+        # Static entity attributes
+        if isinstance(tracker, Tracker01):
+            self._attr_icon = MDI_ICONS[tracker.tracker_config.icon]
+            self._attr_name = self._tracker.name
+            self._attr_unique_id = str(self._tracker.id)
+
+        self.entity_description = GpsTrackerEntityDescription(
+            entity_registry_enabled_default=True,
+            has_entity_name=True,
+            key="invoxia_gps_tracker",
+            translation_key="invoxia_gps_tracker",
+            icon=self._attr_icon,
+        )
+
+        # Dynamic entity attributes
+        self._attr_available: bool = True
+
+        # Dynamic tracker-entity attributes
+        self._tracker_data = GpsTrackerData(
+            latitude=0.0,
+            longitude=0.0,
+            accuracy=0,
+            battery=0,
+        )
+        self._update_attr()
+
+    def _update_attr(self) -> None:
+        """Update dynamic attributes."""
+        LOGGER.debug("Updating attributes of Tracker %u", self._tracker.id)
+        self._tracker_data = self.coordinator.data
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        self._update_attr()
+        return super()._handle_coordinator_update()
+
+    @property
+    def battery_level(self) -> int | None:
+        """Return tracker battery level."""
+        return self._tracker_data.battery
+
+    @property
+    def source_type(self) -> str:
+        """Define source type as being GPS."""
+        return device_tracker.SourceType.GPS
+
+    @property
+    def location_accuracy(self) -> int:
+        """Return accuration of last location data."""
+        return self._tracker_data.accuracy
+
+    @property
+    def latitude(self) -> float | None:
+        """Return last device latitude."""
+        return self._tracker_data.latitude
+
+    @property
+    def longitude(self) -> float | None:
+        """Return last device longitude."""
+        return self._tracker_data.longitude
```

Comment:
It seems to me that these attributes do not have short notations for `device_tracker.TrackerEntity` derivatives.

This is also specified in [current documentation](https://developers.home-assistant.io/docs/core/entity/device-tracker#properties-1): `TrackerEntity does not support attribute shorthand for [property implementation](https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes)` 

When I set `_attr_battery_level` instead of defining the member method `battery_level`, the battery level dissapears from the GUI.
Similarly, if I migrate all attributes to `_attr_*` shorthand, errors are raised when starting the HA instance.

Reply:
Was expecting that all entity platform has support for it. Leave it as it is